backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/chipgroup/multiple/internal/BpkMultiSelectChipGroupImpl.kt

```diff
@@ -84,7 +87,7 @@ internal fun BpkMultiSelectChipGroupImpl(
 
         BpkMultiChipGroupType.Wrap -> {
             FlowRow(
-                modifier = modifier.selectableGroup(),
+                modifier = modifier.selectableGroup().padding(contentPadding),
```

Comment:
technically no `contentPadding`, but the alternative was making the `contentPadding` a param of the rail type which felt weird too or ignoring this